jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/MimeTypes.java

```diff
@@ -797,6 +846,84 @@ else if (!Objects.equals(value, otherValue))
         }
     }
 
+    /**
+     * @deprecated Create a new {@link MimeTypes} with your settings, and call {@link #setFrom(MimeTypes)} instead.
+     */
+    @Deprecated(since = "12.1.0", forRemoval = true)
+    public static class Wrapper extends Mutable
```

Comment:
This class should now also override all the new methods introduced in `MimeTypes` and `Mutable`.